CMakeLists.txt

```
@@ -1,11 +1,38 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.15.0)
 
+ENABLE_LANGUAGE(Fortran)
 PROJECT(MultiphaseStokes LANGUAGES C CXX Fortran)
 FIND_PACKAGE(IBAMR REQUIRED)
 
 SET(CXX_SRC
+    VCTwoFluidStaggeredStokesOperator.cpp
+    VCTwoFluidStaggeredStokesBoxRelaxationFACOperator.cpp
     )
-ADD_EXECUTABLE(main2d main.cpp)
-TARGET_SOURCES(main2d PRIVATE ${CXX_SRC})
-TARGET_LINK_LIBRARIES(main2d IBAMR::IBAMR2d)
-CONFIGURE_FILE(input2d ${CMAKE_CURRENT_BINARY_DIR}/input2d COPYONLY)
+
+SET(F_SRC
+    RBGS.f
+    )
+
+ADD_EXECUTABLE(apply2d apply.cpp)
+TARGET_SOURCES(apply2d PRIVATE ${CXX_SRC})
```

Suggest removing the `${CXX_SRC}` variable because it also requires on `$F_SRC` to be linked correctly. Just list the .cpp and .f files that need to be compiled for each executable.
```suggestion
TARGET_SOURCES(apply2d PRIVATE VCTwoFluidStaggeredStokesOperator.cpp)
```